paasta_tools/marathon_dashboard.py

```diff
@@ -91,7 +91,7 @@ def create_marathon_dashboard(
     dashboard_links = system_paasta_config.get_dashboard_links()
     marathon_links = dashboard_links.get(cluster, {}).get("Marathon RO")
 
-    # e.g. 'http://10.64.97.75:5052': 'http://marathon-norcal-prod.yelpcorp.com'
+    # e.g. 'http://10.64.97.75:5052': 'http://marathon-pnw-prod.yelpcorp.com'
```

Comment:
IP is wrong here too, but I don't think it particularly matters

Reply:
you got me there :p

not sure how worth it is actually getting an ip that would be right if we were still running marathon tho (this file really should just be deleted alongside all the other marathon stuff)